### ISSUE TITLE
Cloudformation: Incorrect/missing outputs when creating a change set

### ISSUE THREAD

**Issue by jeremietharaud** [spulec/moto#3032]
**moto version:**
1.3.15 (spulec/moto: master)

**module:**
mock_cloudformation

**action:**
`create_change_set`
When we create a change set and describe it using the following code, the result is not the same as running using boto3:
```
dummy_template = {
    "AWSTemplateFormatVersion": "2010-09-09",
    "Description": "Stack 1",
    "Resources": {
        "BucketResource": {
            "Type": "AWS::S3::Bucket",
            "Properties": {
                "BucketName": "moto-test-123456789",
            },
        }
    },
}

dummy_template_json = json.dumps(dummy_template)


@mock_cloudformation
def test_change_set():
    cf_conn = boto3.client("cloudformation", region_name="eu-west-1")
    cf_conn.create_change_set(
        StackName="NewStack",
        TemplateBody=dummy_template_json,
        ChangeSetName="NewChangeSet",
        ChangeSetType="CREATE",
    )

    print(cf_conn.describe_change_set(ChangeSetName="NewChangeSet"))


if __name__ == '__main__':
    test_change_set()
```

**expected behavior:**
```
aws cloudformation describe-change-set --change-set-name NewChangeSet --stack-name NewStack
{
    "Changes": [
        {
            "Type": "Resource",
            "ResourceChange": {
                "Action": "Add",
                "LogicalResourceId": "BucketResource",
                "ResourceType": "AWS::S3::Bucket",
                "Scope": [],
                "Details": []
            }
        }
    ],
    "ChangeSetName": "NewChangeSet",
    "ChangeSetId": "arn:aws:cloudformation:eu-west-1:664685189390:changeSet/NewChangeSet/ae79ab51-a4a8-4f4f-8306-af6117f2a1ef",
    "StackId": "arn:aws:cloudformation:eu-west-1:664685189390:stack/NewStack/0b05cf30-a0e0-11ea-9051-0aa5f1c3ad6c",
    "StackName": "NewStack",
    "Description": "Stack 1",
    "Parameters": null,
    "CreationTime": "2020-05-28T12:37:50.669Z",
    "ExecutionStatus": "AVAILABLE",
    "Status": "CREATE_COMPLETE",
    "StatusReason": null,
    "NotificationARNs": [],
    "RollbackConfiguration": {},
    "Capabilities": [],
    "Tags": null
}
```
- Status should be 'CREATE_COMPLETE' (should be fixed by PR #3033)
- Execution status should be 'AVAILABLE' (should be fixed by PR #3033)
- Creation time should be set to the datetime.utcnow() (should be fixed by PR #3033)
- Stack Id should contain the same region in its name as the change set itself, not us-east-1 (should be fixed by PR #3033)
- Changes are missing

**actual behavior:**
```
{
  'ChangeSetName': 'NewChangeSet',
  'ChangeSetId': 'arn:aws:cloudformation:eu-west-1:123456789:changeSet/NewChangeSet/3b0423d6-7514-4b50-91cd-46856bd251de',
  'StackId': 'arn:aws:cloudformation:us-east-1:123456789:stack/NewStack/f62e9101-95c5-4967-a704-fc62190d87ee',
  'StackName': 'NewStack',
  'Description': 'Stack 1',
  'Parameters': [
    
  ],
  'CreationTime': datetime.datetime(2011, 5, 23, 15, 47, 44, tzinfo=tzutc()),
  'ExecutionStatus': '',
  'Status': 'REVIEW_IN_PROGRESS',
  'StatusReason': '',
  'NotificationARNs': [
    
  ],
  'ResponseMetadata': {
    'HTTPStatusCode': 200,
    'HTTPHeaders': {
      'server': 'amazon.com'
    },
    'RetryAttempts': 0
  }
}
```

**Comment by jeremietharaud** [comment]
PR #3033 adds some improvements. Will raise some other for fixing the missing changes during creation of a change set.

**Comment by bblommers** [comment]
This is now available as of moto >= 1.3.15.dev881
